<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="5" hostname="rastkonosovic" failures="1" tests="6" name="D:/Kursevi/REST Assured/Petstore/src/test/java/apiTest" time="6.946" errors="0" timestamp="2023-01-16T09:51:22 CET">
  <testcase classname="apiTest.PetTest" name="testPostPet" time="2.973"/>
  <testcase classname="apiTest.StoreTest" name="testPostOrder" time="0.738"/>
  <testcase classname="apiTest.UserTest" name="testPostUser" time="0.696"/>
  <testcase classname="apiTest.UserTest" name="getUserByName" time="0.682"/>
  <testcase classname="apiTest.UserTest" name="updateUserByName" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at apiTest.UserTest.updateUserByName(UserTest.java:63)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
    </failure>
  </testcase> <!-- updateUserByName -->
  <testcase classname="apiTest.UserTest" name="deleteUserByName" time="0.686"/>
  <testcase classname="apiTest.PetTest" name="updatePetByID" time="0.0">
    <ignored/>
  </testcase> <!-- updatePetByID -->
  <testcase classname="apiTest.StoreTest" name="deleteOrderByID" time="0.0">
    <ignored/>
  </testcase> <!-- deleteOrderByID -->
  <testcase classname="apiTest.PetTest" name="deletePetByID" time="0.0">
    <ignored/>
  </testcase> <!-- deletePetByID -->
  <testcase classname="apiTest.PetTest" name="getPetByID" time="0.0">
    <ignored/>
  </testcase> <!-- getPetByID -->
  <testcase classname="apiTest.StoreTest" name="getOrderByID" time="0.0">
    <ignored/>
  </testcase> <!-- getOrderByID -->
</testsuite> <!-- D:/Kursevi/REST Assured/Petstore/src/test/java/apiTest -->
